LibreNMS/Authentication/LdapAuthorizationAuthorizer.php

```diff
@@ -134,8 +134,8 @@ public function getUserlevel($username)
         // Loop the list and find the highest level
         foreach ($entries as $entry) {
             $groupname = $entry['cn'][0];
-            if (Config::get('auth_ldap_groups')[$groupname]['level'] > $userlevel) {
-                $userlevel = Config::get('auth_ldap_groups')[$groupname]['level'];
+            if (Config::get('auth_ldap_groups.' . $groupname . '.level') > $userlevel) {
```

Comment:
You can't do this because some group names have periods in the names :)

Reply:
Seems like there are a couple of more I need to fix, php 5.3 does not like the array acces directly on the return of a function.
https://3v4l.org/hcOHo#output

Reply:
@murrant is it ok to just make a temp var for it right now?
```
            $authLdapGroups = Config::get('auth_ldap_groups');
            if ($authLdapGroups[$groupname]['level'] > $userlevel) {
                $userlevel = $authLdapGroups[$groupname]['level'];
            }
```
Not sure I want to drag new methods into those classes as part of this refactoring.
The cleanup of the ldap and AD duplication can be for an other refactoring.

Reply:
Yes, that is the best way to fix for now.